operator/Dockerfile.osci

```diff
@@ -0,0 +1,72 @@
+# Deprecated: This is used only by OSCI.
```

Comment:
Something that often gets overlooked: `Dockerfile` is not a file name, it is actually an extension, just without leading dot :-)
IDEs enable syntax highlighting (at least GoLand does so) by extension and so you get highlighting for `Dockerfile` and `osci.Dockerfile` (yes, automatically) but not for `Dockerfile.osci`.

Reply:
Good point, although not so important if this file will disappear this week I guess...